File: drm-core/src/main/kotlin/domain/Group.kt
package moe._47saikyo.domain

data class Group(
    var id: Long = 0,
    var groupName: String? = null,
    var permissionLogin: Boolean = false,
    var permissionShowProfile: Boolean = false,
    var permissionCreateChainAccount: Boolean = false,
    var permissionCreateRight: Boolean = false,
    var permissionVerifyRight: Boolean = false,
    var permissionCreateLicense: Boolean = false,
    var permissionVerifyLicense: Boolean = false,
)

File: drm-webui/src/modules/Group.ts
import Constant from "~/constant/Constant.ts";

class Group {
    id: number = Constant.Global.NULL_NUMBER_PLACEHOLDER;
    groupName: string = Constant.Global.NULL_STRING_PLACEHOLDER;
    permissionLogin: boolean = false;
    permissionShowProfile: boolean = false;
    permissionCreateChainAccount: boolean = false;
    permissionCreateRight: boolean = false;
    permissionVerifyRight: boolean = false;
    permissionCreateLicense: boolean = false;
    permissionVerifyLicense: boolean = false;

}

export default Group